src/style.css

```diff
@@ -1,3 +1,72 @@
+html {
+  margin: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  background-color: white;
+  margin: 0;
+  justify-content: space-between;
+}
+
+.logo {
+  margin: 10px 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__header {
+  display: flex;
+  align-self: flex-end;
+  margin-right: 50px;
+}
+
+.ul__links {
+  margin: 0;
+  display: flex;
+}
+
+.nav__link {
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-decoration: none;
+  margin-left: 20px;
+  height: 60px;
+  text-align: center;
+  line-height: 60px;
```

Comment:
why you need this line-height? just give a vertical paddings in this case

Reply:
for make text center and save clickable zone on 100% of header height

Reply:
that makes sense but it's not the best solution, if you'll won't have space in navigation than on of the links will wrap to the next line and with line-height it will look pretty bad